tests/base/vectorization_07.output

```diff
@@ -0,0 +1 @@
+OK
```

Comment:
Nitpicking: Please rename the test into `vectorization_07.with_cxx11=on.output` to only run it with C++11.